src/main/java/org/broadinstitute/hellbender/tools/ApplyBQSR.java

```diff
@@ -10,8 +10,10 @@
 import org.broadinstitute.hellbender.engine.FeatureContext;
 import org.broadinstitute.hellbender.engine.ReadWalker;
 import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.exceptions.UserException;
```

Comment:
I know this is off topic, but we are going to remove "hellbender" from the path before the alpha is released, correct?